### PR TITLE
Mention helpdesk tickets for score entry

### PR DESCRIPTION
**Pull request by Adimote**
Mention helpdesk tickets for score entry. I'm explicitly not mentioning how or when marshals make notes, because I need to check with Alistair, and that'll likely be a separate PR.

### CODE REVIEW

File: docs/competition/matches/match-scoring.md [hunk 1]
```diff
@@ -101,4 +101,10 @@ The steps to perform are as follows:
 
 There are two score sheets for every match. It's likely that there may be a discrepancy between the sheets. If that's the case, one match score entry volunteer should speak to the marshals to try to find the correct state.
 
-If you have any questions, speak to the head match scorer. In SR2019 the head match scorer is on the balcony.
+If you have any questions, speak to the head match scorer. In SR2023 the head match scorer can be called down from the balcony.
```
Comment (samjmartin04) [inline docs/competition/matches/match-scoring.md:104]: Who is the head match scorer? In SR2022, we didn't have a head match scorer due to the change in how match ops worked. Is this more referring to myself/Alistair now?

Comment (Adimote) [reply docs/competition/matches/match-scoring.md]: Good point, have changed it to the Judge/Competition Software Co-ordinator, based  on the issue.

File: docs/competition/matches/match-scoring.md [hunk 2]
```diff
@@ -101,4 +94,10 @@ The steps to perform are as follows:
 
 There are two score sheets for every match. It's likely that there may be a discrepancy between the sheets. If that's the case, one match score entry volunteer should speak to the marshals to try to find the correct state.
 
-If you have any questions, speak to the head match scorer. In SR2019 the head match scorer is on the balcony.
+If you have any questions, speak to the competition software co-ordinator, or the head judge if it is a judging decision. The head judge should be reachable by radio, the competition software co-ordinator is usually on the balcony.
```
Comment (samjmartin04) [inline docs/competition/matches/match-scoring.md:97]:
```suggestion
If you have any questions, speak to the competition software co-ordinator, or the head judge if it is a judging decision. The competition software co-ordinator should be reachable by radio and the head judge is frequently in the cube.
```
The head judge sadly doesn't have a radio this year

**Review by raccube — APPROVED** [review]
Assuming they have the time to write up tickets, this could work quite well.

**Pull request merged**